src/gt4py/frontend/defir_to_gtir.py

```diff
@@ -81,6 +85,171 @@ def _make_literal(v: numbers.Number) -> gtir.Literal:
         value = str(v)
     return gtir.Literal(dtype=dtype, value=value)
 
+class UnVectorisation(IRNodeVisitor):
+    @classmethod
+    def apply(cls, root, **kwargs):
+        return cls().visit(root, **kwargs)
+
+    def visit_StencilDefinition(self, node: StencilDefinition, fields_decls) -> gtir.Stencil:
```

Comment:
```suggestion
    def visit_StencilDefinition(self, node: StencilDefinition, *, field_decls, **kwargs) -> gtir.Stencil:
```
Please use keyword arguments for all additional parameters as you are breaking the api otherwise. All keyword arguments that you do not consume should be forwarding alla `self.visit(some_thing, **kwargs)`. The same applies to all other `IRNodeVisitors` you introduce. Additional please add a type annotation for `field_decls`.